README.md

```diff
@@ -1,79 +1,152 @@
+                     0.998 R²   (0.996 R² .. 0.999 R²)
+mean                 734.6 μs   (727.6 μs .. 747.0 μs)
+std dev              35.35 μs   (27.33 μs .. 45.79 μs)
+variance introduced by outliers: 40% (moderately inflated)
+                                   
+benchmarking Parse Benchmarks/parseExpr, complexity 80
+time                 1.359 ms   (1.235 ms .. 1.483 ms)
+                     0.960 R²   (0.948 R² .. 0.988 R²)
+mean                 1.241 ms   (1.200 ms .. 1.288 ms)
+std dev              153.8 μs   (112.2 μs .. 205.7 μs)
+variance introduced by outliers: 79% (severely inflated)
+                                   
+benchmarking Parse Benchmarks/parseExpr, complexity 160
+time                 3.153 ms   (2.627 ms .. 3.574 ms)
+                     0.911 R²   (0.865 R² .. 0.959 R²)
+mean                 2.758 ms   (2.627 ms .. 2.918 ms)
+std dev              506.0 μs   (428.3 μs .. 703.7 μs)
+variance introduced by outliers: 88% (severely inflated)
+                                   
+benchmarking Eval Benchmarks/evalExpr, complexity 20
+time                 421.3 ns   (408.0 ns .. 437.0 ns)
+                     0.988 R²   (0.981 R² .. 0.993 R²)
+mean                 458.0 ns   (432.0 ns .. 490.3 ns)
+std dev              98.63 ns   (59.97 ns .. 138.2 ns)
+variance introduced by outliers: 98% (severely inflated)
+                                   
+benchmarking Eval Benchmarks/evalExpr, complexity 40
+time                 927.7 ns   (835.3 ns .. 1.017 μs)
+                     0.912 R²   (0.878 R² .. 0.951 R²)
+mean                 1.065 μs   (979.2 ns .. 1.193 μs)
+std dev              363.1 ns   (290.0 ns .. 470.5 ns)
+variance introduced by outliers: 99% (severely inflated)
+                                   
+benchmarking Eval Benchmarks/evalExpr, complexity 80
+time                 1.413 μs   (1.288 μs .. 1.587 μs)
+                     0.880 R²   (0.826 R² .. 0.937 R²)
+mean                 1.850 μs   (1.686 μs .. 2.159 μs)
+std dev              728.4 ns   (542.0 ns .. 969.8 ns)
+variance introduced by outliers: 99% (severely inflated)
+                                   
+benchmarking Eval Benchmarks/evalExpr, complexity 160
+time                 152.9 ns   (146.0 ns .. 163.8 ns)
+                     0.923 R²   (0.876 R² .. 0.958 R²)
+mean                 204.5 ns   (183.1 ns .. 226.4 ns)
+std dev              80.09 ns   (64.94 ns .. 100.2 ns)
+variance introduced by outliers: 100% (severely inflated)
+```
+Expressions of varying complexity were generated. For every generated `expr`, `(parseExpr . show) expr` and `evalExpr varMap expr` were run.
+Results show that parsing speed is approximately linear from the size of input (which is expected), while evaluation speed is highly unpredictable (which can be seen by variance introduced by outliers).
+Anyway, parsing and evaluation speed are approximately the same order of magnitude, with parsing generally being more costly.
```

Comment:
Parser's performance is usually superlinear, while evaluation is linear, and the size of the input is smaller than that of the parser... Seems very reasonable to me.